sensitivity.tex

```diff
@@ -53,24 +53,35 @@ \subsection{Breeding Ratio}
     \caption{Increasing the breeding ratio decreases the number of total \gls{UOX} \gls{UNF}
              required. The \glspl{ASTRID} produce more plutonium, reducing the plutonium influx
              from reprocessed \gls{UOX}.}
-    \label{fig:eu_tail}
+    \label{fig:br_uox}
 \end{figure}
 
 
 \subsection{Lifetime Extension of French \glspl{LWR}}
 Extending the lifetime of French \glspl{LWR} dramatically lowers the average
 monthly \gls{UOX} reprocessing demand, since the \gls{ASTRID} deployment becomes less
 aggressive. The plutonium demand is delayed allowing the reprocessing plant more
-time to produce plutonium for \gls{ASTRID} reactors. The
+time to prepare plutonium for \gls{ASTRID} reactors. The
 extended deployment schemes are illustrated in figure \ref{fig:ext}. 
 
+Figure \ref{fig:ext_uox} shows the decrease in the average monthly
+\gls{UOX} reprocessing burden with increased lifetime of \glspl{LWR},
+which reduces to the current capacity of the La Hague site if all the
+French \glspl{LWR} extended their operation for 20 years.
+Figure \ref{fig:ext_all} shows that lifetime extension has little
+effect on the average total monthly reprocessing demand, because
```

Comment:
I feel like this is exactly the opposite conclusion from the conclusion drawn in the previous sentence (from the previous figure).

Reply:
I understand how this could be confusing.
`Average uox reprocessing demand` is just the demand for reprocessing legacy UOX 
for the startup of these ASTRIDs,
and `Average total reprocessing demand` is the average from 2040-2160 of the entire
reprocessing demand.
Extending the LWR lifetimes, thus delaying ASTRID deployment,
allows a less rigorous spent UOX reprocessing because the initial deployment isn't so
aggressive. However, the `Average Total` doesn't change much because the plutonium demand
and breeding ratio remains the same. 
Does this explain it? I'm pretty sure there's gotta be a better way to phrase this..